app/middlewares/checkAuthentication.js

```diff
@@ -1,19 +1,21 @@
 import {createError} from '../libs/common';
 import {verifyToken} from '../libs/jwt';
+import httpStatuses from '../constants/httpStatuses';
 
 
 const checkAuthentication = async (req, res, next) => {
+  const {unauthorized} = httpStatuses;
   try {
     const token = req.cookies.token;
     const decodedToken = token && await verifyToken(token);
     if (decodedToken || req.path.includes('/auth/github')) {
       req.user = decodedToken;
       return next();
     } else {
-      return next(createError('Unauthorized', 401));
+      return next(createError(unauthorized.message, unauthorized.statusCode));
```

Comment:
You are trying to incapsulate the same error as in a global catch statement. It would be great to do it once or use another global error